buildpack/telemetry/telegraf.py

```diff
@@ -75,6 +75,8 @@ def _get_config_file_path(version):
 
 
 def include_db_metrics():
+    if metrics.deny_all_apm_metrics():
```

Comment:
This could be put in the same condition below, but I understand keeping the free apps in a separate logic.
So I'm fine with both.

Reply:
yeah, I just wanted to make it very explicit and separate in the beginning since this env var precedes all the other logic